lib/tizen_plugins.dart

```diff
@@ -629,3 +633,82 @@ Future<void> renderTemplateToFile(
   await file.create(recursive: true);
   await file.writeAsString(renderedTemplate);
 }
+
+/// See: [_writeFlutterPluginsList] in flutter_plugins.dart
+Future<void> _writeTizenPluginsExtraInfo(
+  FlutterProject project,
+) async {
+  YamlMap? packagesFromPubspecLock() {
+    final File pubspec = project.directory.childFile('pubspec.lock');
+    if (!pubspec.existsSync()) {
+      return null;
+    }
+
+    Object? contents;
+    try {
+      contents = loadYaml(pubspec.readAsStringSync());
+    } on YamlException catch (err) {
+      globals.printTrace('Failed to parse packages from pubspec.lock: $err');
+    }
+    if (contents == null || contents is! YamlMap) {
+      return null;
+    }
+    final Object? packages = contents['packages'];
+    if (packages == null || packages is! YamlMap) {
+      return null;
+    }
+    return packages;
+  }
+
+  final File appDepsJson = project.directory.childFile('app.deps.json');
+
+  final List<TizenPlugin> plugins = await findTizenPlugins(project);
+  final List<Map<String, Object>> pluginInfo = <Map<String, Object>>[];
+  final YamlMap? packages = packagesFromPubspecLock();
+  for (final TizenPlugin plugin in plugins) {
+    String version = '';
+    if (packages != null && packages.containsKey(plugin.name)) {
+      final YamlMap package = packages[plugin.name] as YamlMap;
+      if (package.containsKey('version')) {
+        version = package['version'] as String;
+      }
+    }
+    pluginInfo.add(<String, Object>{
+      'name': plugin.name,
+      'version': version,
+    });
+  }
+
+  final Map<String, Object?> result = <String, Object>{};
+  result['info'] =
+      'This is a generated file; do not edit or check into version control.';
+  result['plugins'] = pluginInfo;
+  final Map<String, Object> dart = <String, Object>{};
+  dart['version'] = globals.flutterVersion.dartSdkVersion;
+
+  final Map<String, Object> flutter = <String, Object>{};
+  flutter['version'] = globals.flutterVersion.frameworkVersion;
+
+  final Map<String, Object> flutterTizen = <String, Object>{};
+  flutterTizen['revision'] = globals.flutterVersion.frameworkRevisionShort;
```

Comment:
I don't think we need to use short hash. See here. https://github.com/flutter-tizen/flutter-tizen/tree/master/bin/internal